### DISCUSSION TITLE
im not sure if its just me.

### DISCUSSION THREAD

**Discussion by lfmarinescu** [valinet/ExplorerPatcher#3147] [Ideas]
it may just be me but I use explorer patcher on windows 11 because I like the windows 10 feel. thing is I cant find the actions and notifications menu anywhere. where is the notification menu? if its not in explorer patcher maybe you could add it? im not sure.

**Comment by pyrates999** [comment]
Notifications is shown on the bottom right of the windows 10 taskbar.  Clicking on it shows the action center.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by plofhaan** [comment]
Verify that Action Center hasn't been disabled in the Windows registry. To do this, run the command:
`
reg query HKCU\Software\Policies\Microsoft\Windows\Explorer /v DisableNotificationCenter
`

### 1. If it reports that the value is 1
It means that Action Center is disabled. To enable it, open an elevated command prompt and run the command:
`
reg delete HKCU\Software\Policies\Microsoft\Windows\Explorer /v DisableNotificationCenter /f
`
The above command should report that the operation was successful. For this change to take effect, either you can restart the computer or you can log off and log back on again. Please note, when I logged off and back on again, the Action Center Icon was still invisible, with just a blank area where the icon should normally appear. I was able to remedy this by restarting explorer (link in the bottom left corner of the ExplorerPatcher settings window).

### 2. If either it reports that it couldn't find the value or it reports that the value is 0
It means that Action Center is enabled (not hidden), in which cause I would suggest to restart the computer to see if that fixes it. If it doesn't, run the `sfc /scannow` and `dism /online /cleanup-image /restorehealth` commands to see if system files have been corrupted and to automatically repair them if needed.
